code/modules/art/statues.dm

```diff
@@ -1,3 +1,5 @@
+#define SCULPT_SOUND_INCREMENT 4
```

Comment:
Could you autodoc this?

Reply:
https://github.com/tgstation/tgstation/pull/72638/commits/25b840b472ca451c556a0a72effe7e7f9a8c9ea5